src/app/header/components/__tests__/TermButtons.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { getCurrentTerm, getCurrentTerms } from '../../../shared/utils/terms';
+import { TermButtons } from '../TermButtons';
+
+const term = getCurrentTerm();
+const terms = getCurrentTerms();
+
+describe('TermButtons', () => {
+  it('turns the button active on click', () => {
```

Comment:
if the other test below this must wait until the button is loaded to check if the appropriate button is active, does that mean this test is running before the button is fully loaded? Seems like that presents a bit of a race condition?